classloader-leak-prevention/src/main/java/se/jiderhamn/classloader/leak/prevention/cleanup/MultiThreadedHttpConnectionManagerCleanUp.java

```diff
@@ -0,0 +1,26 @@
+package se.jiderhamn.classloader.leak.prevention.cleanup;
+
+import se.jiderhamn.classloader.leak.prevention.ClassLoaderLeakPreventor;
+import se.jiderhamn.classloader.leak.prevention.ClassLoaderPreMortemCleanUp;
+
+/**
+ * Invokes static method org.apache.commons.httpclient.MultiThreadedHttpConnectionManager.shutdownAll() to close connections left out by com.sun.jersey.client.apache.ApacheHttpClient.  
+ * 
+ * @author Marian Petrik
+ */
+public class MultiThreadedHttpConnectionManagerCleanUp implements ClassLoaderPreMortemCleanUp {
+	
+  @Override
+  public void cleanUp(ClassLoaderLeakPreventor preventor) {
+	    final Class<?> connManager = preventor.findClass("org.apache.commons.httpclient.MultiThreadedHttpConnectionManager");
+	    if(connManager != null) {
+	      try {
+	    	  connManager.getMethod("shutdownAll").invoke(null);
```

Comment:
I though `ClassLoaderLeakPreventor` already had an `invokeStaticMethod(Class clazz, String name)`, but this seems like a good time to create one :-)